usaspending_api/download/helpers/elasticsearch_download_functions.py

```diff
@@ -67,6 +68,8 @@ def _get_download_ids(cls, filters: dict, size: int = 10000) -> QuerySet:
         ids = cls._get_download_ids_generator(search, size)
         flat_ids = list(itertools.chain.from_iterable(ids))
         logger.info(f"Found {len(flat_ids)} {cls._source_field} based on filters")
```

Comment:
I think the original logging statement is redundant with the `write_to_log` call and can be removed